Test/Operator/test_operator.py

```diff
@@ -34,7 +34,7 @@
 
 def test_produce_elements_in_hilbert():
     for name, ha in operators.items():
-        hi = ha.GetHilbert()
+        hi = ha.get_hilbert()
```

Comment:
It is still a mistery to me why here get_hilbert works without seg_faulting, whereas in machine.get_hilbert breaks down. Both return references and not pointers.... @jamesETsmith @femtobit @twesterhout

Reply:
@gcarleo you might want to study the sections on return value policies: https://pybind11.readthedocs.io/en/stable/advanced/functions.html#return-value-policies. They explain stuff way better than I ever could.